### PR TITLE
Remove operation handling and add recipes

### PR DESCRIPTION
**Pull request by petter-fidesmo**
- Add recipe for install
- Add recipe for delete

### CODE REVIEW

**Comment by petter-fidesmo** [comment]
@sergkh @pepegar Ready for review

File: src/main/groovy/com/fidesmo/gradle/plugin/RecipeTask.groovy [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+           .setErrorHandler(
+               new ErrorHandler(){
+                   Throwable handleError(RetrofitError cause) {
+                       if (cause.isNetworkError()) {
+                           new GradleException('An network related error occured while uploading the cap file', cause)
+                       } else {
+                           try {
+                               if (cause.response.status >= 500) {
+                                   def errorMessage = cause.response.body.in().text
+                                   new GradleException("The fidemo server aborted the operation with '${errorMessage}'", cause)
```
Comment (pepegar) [inline src/main/groovy/com/fidesmo/gradle/plugin/RecipeTask.groovy]: Typo here, `Fidesmo` :D

**Comment by sergkh** [comment]
A bit offtopic, but probably we should also change register in the `SmartcardioCard` because I always read it as Smart Cardio. 

Or it is  am I the only one like that?

Naming it like `SmartCardIoCard` would help

**Comment by petter-fidesmo** [comment]
@sergkh I agree. I think it was named after javax smartcardio extension, but I renamed it now. We don't have to make the same mistake as them :-).

**Pull request merged**